### ISSUE TITLE
The definitions of several transformation and frame.

### ISSUE THREAD

**Issue by SangbaekYoo** [eigenvivek/DiffPose#44]
Hello! 
I'm trying to understand some code, but I'm not clear on how certain transformations and frame are precisely defined, so I wanted to ask a few questions.

1. Camera frame
Generally, camera frame is defined as optical axis-> z-axis, image height axis -> y-axis, image width axis -> x-axis.
But in the medical registration problem, it may not. Is there any clear definition of camera frame?

2. flip_xz
When you use ljubljana dataset, you define these transformations.
```
self.flip_xz = RigidTransform(
            torch.tensor(
                [
                    [0.0, 0.0, -1.0],
                    [0.0, 1.0, 0.0],
                    [1.0, 0.0, 0.0],
                ]
            ),
            torch.zeros(3),
        )
```
```
self.translate = RigidTransform(
            torch.eye(3),
            torch.tensor([-self.focal_len / 2, 0.0, 0.0]),
        )
```
and when you calculate the camera frame, you implemented as follows
```
extrinsic = (
            self.flip_xz.inverse().compose(self.translate.inverse()).compose(pose)
        )
```
Why these transformations are needed?

Maybe there are general concept, but I could not find these definitions.

**Comment by SangbaekYoo** [comment]
I reviewed the paper and the code and understood what they mean. Thank you!
